src/puppetlabs/puppetdb/utils.clj

```diff
@@ -343,6 +345,29 @@
     {:command command :version (Integer. version)
      :certname certname :payload (json/->RawJsonString payload)}))
 
+(defn utf8-bytes [s] (.getBytes ^String s "UTF-8"))
+
+(defn utf8-length
+  "Return the length in bytes of the given string when encoded in UTF-8"
+  [s]
+  (count (.getBytes ^String s "UTF-8")))
+
+(def ^Charset utf8-charset (Charset/forName "UTF-8"))
```

Comment:
Might just grab it from here instead: https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html